nowplaying/input/handler.py

```diff
@@ -22,12 +22,12 @@ def __init__(self):
         self.__observers = []
         self.last_input = 1
 
-        self.saemubox = misc.saemubox.SaemuBox()
+        self.saemubox = SaemuBox()
 
     def register_observer(self, observer):
-        if not isinstance(observer, input.observer.InputObserver):
-            raise Exception("Only InputObserver objects can be registered")
-
+        # if not isinstance(observer, InputObserver):
```

Comment:
What's the reason for commenting (disabling) the instance check?

Reply:
`nowplaying/track` already had it disabled (in exactly this way) and I'm going to refactor it into an ABC with proper type hinting down the line. In fact this refactor to a proper ABC is already prepared on a branch that I will rebase once this PR is merged.

Reply:
I don't see what's broken with the test and why it has to be disabled. The refactoring sounds like a good plan, until then we can leave the type check in place, unless there's an actual problem with it.

Reply:
I would not have commented it if it wasn't broken in some way. The exact same commenting pattern was already in nowplaying/track. The ABC refactor is already done and just waiting for #30 and #31 to get merged so I can rebased it onto this.

Reply:
It probably didn't work anymore, because you changed the import statements at the beginning... But anyway, it certainly did work, as it is active in the current running version, I lost track about who did whatever changes for whatever reasons...

Reply:
This is getting refactored to use an ABC and type hinting in #35 
https://github.com/radiorabe/nowplaying/blob/c17f85ff1d1db58d855d1fa3eb51ea0acfe93f6a/nowplaying/input/handler.py#L22-L24